__device-tests__/gutenberg-editor-sanity-test-1-visual.test.js

```diff
@@ -0,0 +1,294 @@
+/**
+ * External dependencies
+ */
+import wd from 'wd';
+
+/**
+ * Internal dependencies
+ */
+const { blockNames } = editorPage;
+import { takeScreenshot } from './utils';
+const { toggleOrientation, isAndroid, swipeFromTo } = e2eUtils;
+
+describe( 'Gutenberg Editor - Test Suite 4', () => {
+	describe( 'Columns block', () => {
+		it( 'displays placeholders when unselected', async () => {
+			await editorPage.addNewBlock( blockNames.columns );
+			// Wait for the modal to open
+			await editorPage.driver.sleep( 3000 );
+			// Dismiss coluns layout picker
```

Comment:
There's a small typo here
```suggestion
			// Dismiss columns layout picker
```